src/InteractiveMars/Mars.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Mars, renderToolTipsOnHover } from './Mars';
+import { Marker } from "react-globe"
+
+test('renders Mars ', () => {
+    const { getByText } = render(<Mars />);
+    const linkElement = getByText(/ /);
+    expect(linkElement).toBeInTheDocument();
+});
+
+test('Tooltip returns info', () => {
+    const marker: Marker = 
+        {
+            id: "1",
+            rover: 'Curiosity',
+            color: 'red',
+            coordinates: [4.5895, 137.4417],
+            value: 35,
+        }
+
+    const output = `ROVER: ${marker.rover}`
+    expect(renderToolTipsOnHover(marker)).toBe(output)
```

Comment:
this test ends up just running the exact same code twice.

ie the test does ``ROVER: ${marker.rover}``, and the real code does ``ROVER: ${marker.rover}``... 
and then we check that they are the same.

That's always going to be true... even if it isn't actually correct.

Could you change this test to just do
```TypeScript
const marker = {
    // the marker stuff
};

const toolTipText = renderToolTopsOnHover(marker);

expect(toolTipText).toBe("ROVER: Curiosity");
```